### PR TITLE
Msg labelling fixes

### PR DESCRIPTION
**Pull request by rowanseymour**
* Don't include test messages in filter by label view
* Don't list label folders in Add Label actions

### CODE REVIEW

File: temba/msgs/tests.py [hunk 1]
```diff
@@ -414,33 +414,6 @@ def test_inbox(self):
         self.assertEqual(list(Msg.objects.filter(labels=label1)), [msg1])
         self.assertEqual(list(Msg.objects.filter(labels=label3)), [msg3, msg2, msg1])
 
-        # Let's test the filter by labels
```
Comment (rowanseymour) [inline temba/msgs/tests.py]: Moved this to it's own `test_filter` method so it's easier to see what exactly is being tested for that page

File: temba/tests.py [hunk 2]
```diff
@@ -135,13 +135,13 @@ def create_group(self, name, contacts):
         return group
 
     def create_msg(self, **kwargs):
-        if not 'org' in kwargs:
+        if 'org' not in kwargs:
```
Comment (rowanseymour) [inline temba/tests.py]: `not x in y` is a PEP warning to use `x not in y`

File: temba/msgs/views.py [hunk 3]
```diff
@@ -807,7 +807,7 @@ def derive_label(self):
 
         def get_queryset(self, **kwargs):
             qs = super(MsgCRUDL.Filter, self).get_queryset(**kwargs)
-            qs = self.derive_label().filter_messages(qs).filter(visibility=VISIBLE)
+            qs = self.derive_label().filter_messages(qs).filter(visibility=VISIBLE, contact__is_test=False)
```
Comment (nicpottier) [inline temba/msgs/views.py]: This is fine, though I wonder whether we should just make labeling of messages in the simulator be a no-op. There isn't any way of testing for the label of a message, nor any way to see the labeling of messages done through the simulator, so might be a significant performance win to not require this join.

**Comment by nicpottier** [comment]
Yep

**Pull request merged**